src/panels/MinimalPanel.vue

```diff
@@ -156,6 +168,14 @@ export default {
     padding: 0;
   }
 
+  .card-container {
+    pointer-events: none;
+  }
+
+  h1, h2, h3, h4, h5, h6 {
```

Comment:
could we make this more specific? ( only for headings in the `header-wrapper` / `morph-display-wrapper` where we actually need to disable it )